docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -238,6 +238,25 @@ otherwise the procedure will fail with similar message:
 ``Retention specified (1.00d) is shorter than the minimum retention configured in the system (7.00d)``.
 The default value for this property is ``7d``.
 
+.. _iceberg-alter-table-set-properties:
+
+ALTER TABLE SET PROPERTIES
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector supports modifying the properties on existing tables using
+:ref:`ALTER TABLE SET PROPERTIES <alter-table-set-properties>`.
+
+The following table properties can be updated after a table is created:
```

Comment:
```
SELECT * FROM system.metadata.table_properties where catalog_name = 'iceberg';
```

Do we / should we have documented somewhere this statement used to retrieve the "updatable" table properties?